app/styles/_core.less

```diff
@@ -503,18 +503,34 @@ label.checkbox {
 }
 
 .osc-secrets-form {
-  .picked-secret {
-    margin-bottom: 5px;
+  .remove-secret {
+    display: inline-block;
+    margin-left: 7px;
+    vertical-align: 0px;
   }
-  .directory {
-    width:90%;
+  .secret-name(@width: 90%) {
```

Comment:
I think I'd rather just have

```css
.secret-name {
  width: 90%;
  display: inline-block;
}
```

and later

```css
.basic-secrets {
  .secret-name {
    width: 95%;
  }
}
```

I don't think the mixin is necessary?